### PR TITLE
[FIX] point_of_sale: Send & Print upon invoice generation

### PR DESCRIPTION
**Pull request by JulienVR**
task-3542881

related-https://github.com/odoo/enterprise/pull/49137

### CODE REVIEW

File: addons/account/wizard/account_move_send.py [hunk 1]
```diff
@@ -590,6 +590,7 @@ def _download(self, attachment_ids):
             }
         else:
             filename = self.move_ids._get_invoice_report_filename(extension='zip') if len(self.move_ids) == 1 else 'invoices.zip'
+            print(url_encode({'ids': attachment_ids, 'filename': filename}))
```
Comment (mart-e) [inline addons/account/wizard/account_move_send.py:593]: oups

**Review by mart-e — COMMENTED** [review]
@robodoo override=ci/security
Reason: zipping attachments

File: addons/account/models/account_move.py [hunk 2]
```diff
@@ -4292,36 +4292,40 @@ def _generate_qr_code(self, silent_errors=False):
...
+            # if the Send & Print succeeded
             return self.invoice_pdf_report_id.raw, self.invoice_pdf_report_id.name
-        else:
-            return self.env['ir.actions.report']._render('account.account_invoices', self.ids)[0], "Invoices.pdf"
+        elif len(self) < 2 and self.message_main_attachment_id:
+            # if the Send & Print failed with fallback=True -> proforma PDF
+            return self.message_main_attachment_id.raw, self.message_main_attachment_id.name
+        # all other cases
+        pdf_content = self.env['ir.actions.report']._render('account.account_invoices', self.ids)[0]
+        pdf_name = "Invoice.pdf" if len(self) == 1 else "Invoices.pdf"
```
Comment (smetl) [inline addons/account/models/account_move.py:4327]: use `_get_invoice_pdf_report_filename` if there is only one invoice plz

**Review by smetl — COMMENTED** [review]
tiny detail

**Comment by smetl** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**